pulpcore/pulpcore/app/viewsets/content.py

```diff
@@ -62,7 +64,11 @@ def create(self, request):
         return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
 
 
-class ArtifactViewSet(CreateDestroyReadNamedModelViewSet):
+class ArtifactViewSet(NamedModelViewSet,
+                      mixins.CreateModelMixin,
+                      mixins.RetrieveModelMixin,
+                      mixins.DestroyModelMixin,
```

Comment:
since we define destroy() we can drop this mixin

Reply:
we can't, because in our definition we call super()